torch/_dynamo/convert_frame.py

```diff
@@ -541,23 +541,24 @@ def __call__(
             info = f"{code.co_name} {code.co_filename}:{code.co_firstlineno}"
             dynamo_tls.traced_frame_infos.append(info)
 
-        return _compile(
-            frame.f_code,
-            frame.f_globals,
-            frame.f_locals,
-            frame.f_builtins,
-            self._torchdynamo_orig_callable,
-            self._one_graph,
-            self._export,
-            self._export_constraints,
-            hooks,
-            cache_entry,
-            cache_size,
-            frame,
-            frame_state=frame_state,
-            compile_id=compile_id,
-            skip=skip + 1,
-        )
+        with compile_context(CompileContext(compile_id)):
```

Comment:
This is unexpected change, not sure if you needed to solve some other problem. Compile context is set slightly later at https://github.com/ppanchalia/pytorch/blob/export-D64863946/torch/_dynamo/convert_frame.py#L853 so you shouldn't need to do it here

Reply:
Ah .. yeah, when repro'ing the original issue, I noticed that this https://github.com/ppanchalia/pytorch/blob/export-D64863946/torch/_dynamo/convert_frame.py#L694 doesn't have the compile context set and was leading to the an extraneous (unknown) in the tlparse while logging the chromium event. I added it to eliminate all such occurrences that surfaced in the repro.

Reply:
OK